actionpack/lib/action_controller/renderer.rb

```diff
@@ -71,6 +71,18 @@ def initialize(controller, env, defaults)
     end
 
     # Render templates with any options from ActionController::Base#render_to_string.
+    #
+    # The primary options are:
+    # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
+    # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
+    # * <tt>:inline</tt> - Renders an inline template.
+    # * <tt>:plain</tt> - Renders provided text and sets the content type as <tt>text/plain</tt>.
+    # * <tt>:html</tt> - Renders the provided HTML safe string, otherwise
```

Comment:
I've used `json` more than I've used `html` because `html` is usually the default right? I would make sure to cover the main options mentioned in the guides. See http://guides.rubyonrails.org/layouts_and_rendering.html#wrapping-it-up

Reply:
Yes, please add Json